spec/factories/disconnect_initiator.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :disconnect_initiator, class: DisconnectInitiator do
+    sequence(:id) { |n| n }
+    name { 'Disconnect initiator' }
+
+    trait :traffic_manager do
+      id { 0 }
+      name { 'Traffic manager' }
+    end
+
+    trait :traffic_switch do
+      id { 1 }
+      name { 'Traffic switch' }
+    end
+
+    trait :destination do
+      id { 2 }
+      name { 'Destination' }
+    end
+
+    trait :Origination do
+      id { 3 }
```

Comment:
will be good to have this magic numbers in constants defined in DisconnectInitiator model

example: ID_ORIGINATION = 3, ID_TRAFFIC_MANAGER = 0, etc